src/lib/storage/storage_manager.cpp

```diff
@@ -38,7 +38,7 @@ void StorageManager::drop_table(const std::string& name) {
 }
 
 std::shared_ptr<Table> StorageManager::get_table(const std::string& name) const {
-  if (Hyrise::get().meta_table_manager.is_meta_table_name(name)) {
+  if (MetaTableManager::is_meta_table_name(name)) {
```

Comment:
Why don't we need the `Hyrise::get()` anymore?

Reply:
Because we are calling a static method. With clang-tidy-9, it start complaining if you call static members through an instance.